float64 sub1
float64 sub2
float64 sub3
---
float64 avg
